client/views/join_channel.tpl

```diff
@@ -0,0 +1,5 @@
+<form id="join-channel-{{id}}" class="join-form" method="post" action="">
+	<input type="text" class="input" name="channel" placeholder="Channel" pattern="[^\s]+" maxlength="200" title="Should be a valid channel name">
```

Comment:
I wondering if we shouldn't add `autocomplete` here. @xPaw?

Reply:
chrome no longer respects `autocomplete` - still worth adding? If so, I'm a big fan.

Reply:
wat, seriously? It's part of the HTML5 spec (as opposed to `autocorrect`), so that's kinda surprising...

Reply:
https://bugs.chromium.org/p/chromium/issues/detail?id=468153